packages/plugin-js-packages/src/lib/runner/utils.ts

```diff
@@ -40,3 +44,12 @@ export function filterAuditResult(
     summary: uniqueResult.summary,
   };
 }
+
+export async function getTotalDependencies(
+  depGroup: DependencyGroup,
+  packageJsonPath: string,
+) {
+  const packageJson = await readJsonFile<PackageJson>(packageJsonPath);
+  const totalDeps = packageJson[dependencyGroupToLong[depGroup]];
+  return totalDeps == null ? 0 : objectToKeys(totalDeps).length;
```

Comment:
The `totalDeps` variable contains an object, not a number, so the name is misleading:

```suggestion
  const deps = packageJson[dependencyGroupToLong[depGroup]];
  return deps == null ? 0 : objectToKeys(totalDeps).length;
```